functests/0_config/config.go

```diff
@@ -130,3 +94,73 @@ var _ = Describe("[performance][config] Performance configuration", func() {
 	})
 
 })
+
+func externalPerformanceProfile(performanceManifest string) (*performancev1alpha1.PerformanceProfile, error) {
+	performanceScheme := runtime.NewScheme()
+	apis.AddToScheme(performanceScheme)
```

Comment:
is it OK to this multiple times, right? Any side effects?

Reply:
Why multiple times? This is happening only once.
In any case the scheme we are adding to is local, used only for the decoder.

Reply:
Nevermind. I was worried about some code calling this code more than once. But this can't happen as things stand today.